kratos/solving_strategies/strategies/explicit_solving_strategy.h

```diff
@@ -0,0 +1,657 @@
+     * @param rModelPart The model part to be computed
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     * @param RebuildLevel The flag to set if the DOF set is rebuild or not
+     */
+    explicit ExplicitSolvingStrategy(
+        ModelPart &rModelPart,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : mrModelPart(rModelPart)
+    {
+        SetMoveMeshFlag(MoveMeshFlag);
+        SetRebuildLevel(RebuildLevel);
+        mpExplicitBuilder = Kratos::make_unique<ExplicitBuilder<TSparseSpace, TDenseSpace>>();
+    }
+
+    /** Copy constructor.
+     */
+    ExplicitSolvingStrategy(const ExplicitSolvingStrategy &Other) = delete;
+
+    /** Destructor.
+     */
+    virtual ~ExplicitSolvingStrategy() = default;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /**
+     * @brief Operation to predict the solution ... if it is not called a trivial predictor is used in which the
+     * values of the solution step of interest are assumed equal to the old values
+     */
+    virtual void Predict()
+    {
+    }
+
+    /**
+     * @brief Initialization of member variables and prior operations
+     */
+    virtual void Initialize()
+    {
+        // Initialize elements, conditions and constraints
+        InitializeContainer(GetModelPart().Elements());
+        InitializeContainer(GetModelPart().Conditions());
+        InitializeContainer(GetModelPart().MasterSlaveConstraints());
+
+        // Set the explicit DOFs rebuild level
+        if (mRebuildLevel != 0) {
+            mpExplicitBuilder->SetResetDofSetFlag(true);
+        }
+
+        // If the mesh is updated at each step, we require to accordingly update the lumped mass at each step
+        if (mMoveMeshFlag) {
+            mpExplicitBuilder->SetResetLumpedMassVectorFlag(true);
+        }
+
+        // Call the explicit builder and solver initialize (Set up DOF set and lumped mass vector)
+        mpExplicitBuilder->Initialize(mrModelPart);
+
+        // Initialize the solution values
+        InitializeDofSetValues();
+    }
+
+    /**
+     * @brief Clears the internal storage
+     */
+    virtual void Clear()
+    {
+        // This clears the DOF set and lumped mass vector
+        mpExplicitBuilder->Clear();
+    }
+
+    /**
+     * @brief This should be considered as a "post solution" convergence check which is useful for coupled analysis
+     * @details The convergence criteria used is the one used inside the "solve" step
+     */
+    virtual bool IsConverged()
+    {
+        return true;
+    }
+
+    /**
+     * @brief This operations should be called before printing the results when non trivial results (e.g. stresses)
+     * need to be calculated given the solution of the step
+     * @details This operations should be called only when needed, before printing as it can involve a non negligible cost
+     */
+    virtual void CalculateOutputData()
+    {
+    }
```

Comment:
are those necessary for the explicit strategy?

Reply:
I'd keep the `IsConverged` because it might be useful to make the upper layer (i.e. Python) solver compatible with both implicit and explicit strategies.
Regarding the second one, I think it wouldn't be used as we normally do those things using the `ExecuteBeforeOutputStep` of the processes. Furthermore I think we are not using it in the already existent strategies, so we can remove it not only here but in all the other strategies.